frontend/src/metabase/query_builder/actions/visualization-settings.ts

```diff
@@ -57,8 +63,3 @@ export const replaceAllCardVisualizationSettings =
       }),
     );
   };
-
```

Comment:
I don't think we need these, all usages seem to have been replaced

Reply:
huh. I think I must have had a bad search then.

Reply:
Wait @heypoom I changed the name of the original function to `onUpdateVisualizationSettings` - we should be checking for any references that still refer to `updateCardVisualizationSettings` or `replaceAllCardVisualizationSettings`

Reply:
no worries 😄 thanks for checking!